.gitignore

```diff
@@ -1,2 +1,2 @@
-
+replication
```

Comment:
Can we also add a `.dockerignore` with this, as otherwise it'll become part of the build context.